exercises/tom_perkins/ex02_oo_basics/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required ( VERSION 2.8.5 )
+
+###############################################################################
+# Setup
+###############################################################################
+# Name
+project ( Shapes )
+
+# Find custom cmake modules
+set ( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../CMake")
+
+# Common setup
+include ( CommonSetup )
+
+###############################################################################
+# Executable
+###############################################################################
+
+set ( SRC_FILES
+  src/main.cpp
+  # add any additional source files here
+  src/rectangle.cpp 
+  src/square.cpp
+  src/circle.cpp 
+  src/triangle.cpp
+  src/shapesorter.cpp 
+  src/typetester.cpp 
+  src/sidestester.cpp 
+)
+
+set ( HDR_FILES
+  # add any additional header files here
+  src/shape.h
+  src/rectangle.h 
+  src/square.h
+  src/circle.h 
+  src/triangle.h   
+  src/shapesorter.h 
+  src/ShapeTester.h
```

Comment:
File naming case is differnt. On Mantid we use Pascal case. Except for python files.